x-pack/plugins/siem/public/components/page/hosts/events_table/index.tsx

```diff
@@ -254,3 +254,24 @@ const getEventsColumns = (
     },
   },
 ];
+
+const getEventsColumnsCurated = (pageType: hostsModel.HostsType) => {
+  const columnsCurated = getEventsColumns(pageType);
+
+  if (pageType === 'details') {
```

Comment:
Again, simplify this `if` and add a test please

Reply:
👍